frontend/second-battle-of-panipat-explorer/style.css

```
@@ -0,0 +1,468 @@
+
+/* Sections — only the active one is shown */
+.sbp-section {
+    display: none;
+    animation: sbpFadeIn 0.4s ease;
+}
+
+.sbp-section.active {
+    display: block;
+}
+
+@keyframes sbpFadeIn {
```

Stylelint (17.14.0)

[error] Expected keyframe name "sbpFadeIn" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)